### ISSUE TITLE
Create mechanism to apply for many project memberships at once

### ISSUE THREAD

**Issue by murphysj** [EarthSystemCoG/COG#229]
WHO: Allyn
Benefits: All Users

It might be good to have a single page where the user can browse all projects, and apply for membership in each of them, in addition to the in-context link displayed on each project page

**Comment by murphysj** [comment]
If we really want users to join projects, there needs to be an easier way of doing it.

**Comment by LucaCinquini** [comment]
I don't agree with this. What would end up happening is that users or robots would visit that page and click every single box to apply to all projects.
